SDK/src/NDK/Sdk.cpp

```diff
@@ -63,18 +64,10 @@ namespace Ndk
 			// Initialize the engine first
 
 			// Shared modules
-			#ifdef NDK_SERVER
-			Nz::ParameterList parameters;
-			parameters.SetParameter("NoWindowSystem", true);
-
-			Nz::Utility::SetParameters(parameters);
-			#endif
-
 			Nz::Lua::Initialize();
 			Nz::Noise::Initialize();
 			Nz::Physics2D::Initialize();
 			Nz::Physics3D::Initialize();
-			Nz::Utility::Initialize();
```

Comment:
I thought that since Graphics was calling Renderer, hence Utility, it was no more required. But the server can handle Mesh or Image too.